model/models.py

```diff
@@ -1,6 +1,8 @@
+from model.board import Board
+
 class MineSweeper:
-    bombs = [(1,1)]
-    size = [(3,3)]
 
-    def has_mine(self, position):
-        return position in MineSweeper.bombs
+    def validate_position(self, position):
+        board = Board()
```

Comment:
The `Board` instance should be saved in the `MineSwepper` instance. (i.e. `self.board = Board()` in the `__init__`)